### ISSUE TITLE
ServiceAccount user getting throttled

### ISSUE THREAD

**Issue by rootMuBlackArrow** [naturalcrit/homebrewery#2322]
Error: Internal Server Error

```
Error: Internal Server Error
    at Request.<anonymous> (https://homebrewery.naturalcrit.com/homebrew/bundle.js:75346:19)
    at Emitter.emit (https://homebrewery.naturalcrit.com/homebrew/bundle.js:1966:20)
    at xhr.onreadystatechange (https://homebrewery.naturalcrit.com/homebrew/bundle.js:75655:10)
{
  "status": 500,
  "method": "PUT",
  "url": "/api/update/rm1LlRb2LlyE"
}
```

**Comment by calculuschild** [comment]
Thank you for the report! We are looking into it.

**Comment by calculuschild** [comment]
Two tasks:

- [ ] Apply the quotaUser parameter if it applies
- [ ] Check for redundant/needless requests to Google

**Comment by calculuschild** [comment]
Graph showing the frequency of UserRateExceeded errors over a 48 hour period, totalling 689 events:

![image](https://user-images.githubusercontent.com/5878534/189023291-2d1e04ba-65b0-40b1-865e-4acf88d26a93.png)

**Comment by calculuschild** [comment]
Fixed by #2470

Back to lower Google API traffic:

![image](https://user-images.githubusercontent.com/5878534/210399009-7a0e77eb-a37a-4c29-8e56-289ba9e8c908.png)
